controllers/pkg/reconcilers/repository/reconciler.go

```diff
@@ -42,9 +41,6 @@ func init() {
 }
 
 const (
-	finalizer = "infra.nephio.org/finalizer"
```

Comment:
we could keep these here since it is good practice to initialise the const. So the motivation is to be apply to copy paste the code below and the dev just has to update the const. The rest will flow.

Reply:
The err isn't used at all so will remove.
The finalizr const is used once so doesn't really require a const but will revert anyway.

Reply:
> we could keep these here since it is good practice to initialise the const. So the motivation is to be apply to copy paste the code below and the dev just has to update the const. The rest will flow.

The err isn't used at all so will remove.
The finalizr const is used once so doesn't really require a const but will revert anyway.

Reply:
yes err can eb removed, but in another controller you just need to change the const at the top

Reply:
> yes err can eb removed, but in another controller you just need to change the const at the top

Understood. Done